Functional annotation and comparative genomics
====

This is the last module of the Microbial Genomics course at the MSc Bioinformatics and Biocomplexity at Utrecht University.

The **learning objectives** in this module are:
- to understand how the function of proteins are predicted
- to learn how to perform typical -omics data analyses to identify candidate genes for further functional analyses
- to think critically about the results obtained with bioinformatics tools
- to find relevant information

This final module of the course is more biological than previous ones and is therefore more prone to variability in the results. The answers to exercises are only for indication; there are other ways to perform the analyses which will yield different results (no better or worse, just different). In each exercise, you must use a critical mind and search for relevant information to decide what you think is sensible from a biological point of view.

** **

**Overview of the module**

*Day 1: 25 Nov, BOL â€“ 1.138 / 1.128*
1. 9h-9h45 Introduction about fungal genomic research and functional annotation
2. 10h-15h Perform functional annotation with different methods
   Exercise 1: annotation using similarity search
   Exercise 2: search for conserved domains
   Exercise 3: search for conserved motifs
   Discussion
3. 15h-16h Lecture about comparative genomics

** **

**Data availability**

On the server B-MBIMIGE2024 ~/data/fungalgenomics_collemare/day1/ and ~/data/fungalgenomics_collemare/day2/

All presentations, assignments, command lines and answers on https://github.com/JeromeCollemare/Microbial_Genomics_Course in due time
